.travis.yml

```diff
@@ -2,9 +2,12 @@ sudo: false
 
 language: go
 
-go:
-  - 1.4
-  - 1.3
+matrix:
+  include:
+    - go: 1.4
+      env: KUBE_TEST_API_VERSIONS="v1beta1"
+    - go: 1.3
+      env: KUBE_TEST_API_VERSIONS="v1beta3"
```

Comment:
was there any particular pattern to using v1beta3 with go1.3 and v1beta1 with go1.4? or were they just semi-randomly chosen?